.github/CODEOWNERS

```
@@ -1,15 +1,15 @@
 /cray/modules/ars       @Cray-HPE/cray-management-systems
 /cray/modules/artifacts @Cray-HPE/cray-management-systems
```

I think PET owns Ceph/S3, right?

```suggestion
/cray/modules/artifacts @Cray-HPE/platform-engineering
```